clang/include/clang/Sema/Sema.h

```diff
@@ -3144,6 +3154,9 @@ class Sema final {
                               QualType T, TypeSourceInfo *TSInfo,
                               StorageClass SC);
 
+  /// Potentially add a FunctionDecl or BlockDecl to DeclsWithEffectsToVerify.
+  void CheckAddCallableWithEffects(const Decl *D, FunctionEffectSet FX);
```

Comment:
Yeah this is a hard area for me with names.

What it's trying to convey is that it might, depending on some checks, add the callable to a list of decls to verify.

Reply:
I’m not particularly good at naming myself, but maybe something like `CheckDeclWithEffects` would be less of a headache to parse. If you can think of a better name, feel free to go with that instead of course.

Reply:
this is now part of the follow-on PR